lib/travis/build/templates/header.sh

```diff
@@ -85,6 +85,22 @@ function travis_nanoseconds() {
   $cmd -u $format
 }
 
+travis_internal_ruby() {
+  if ! type rvm &>/dev/null; then
+    source ~/.rvm/scripts/rvm &>/dev/null
+  fi
+
+  local internal_ruby
+  internal_ruby="$(
+    rvm list \
+      | awk -F- '/ruby-(2\.[0-2]\.[0-9]|1\.9\.3)/ { print $2 }' \
```

Comment:
The regex matches OK, but `sort -n` means that `2.1.5` is "greater than" `2.1.10`.